graylog2-server/src/main/java/org/graylog2/configuration/ElasticsearchClientConfiguration.java

```diff
@@ -54,4 +54,7 @@
 
     @Parameter(value = "elasticsearch_discovery_frequency")
     private Duration discoveryFrequency = Duration.seconds(30L);
+
+    @Parameter(value = "elasticsearch_compression_enabled")
+    private boolean compressionEnabled = false;
```

Comment:
Please add this setting (and some explaining comments) to the example configuration file in `misc/graylog.conf`.